website/docusaurus.config.ts

```diff
@@ -17,6 +17,14 @@ module.exports = async function (): Promise<Config> {
         organizationName: "Authentik Security Inc.",
         projectName: "authentik",
         themeConfig: {
+            announcementBar: {
+                id: "support_us",
+                content:
+                    'Change is hard. We hate it too when a familiar site or application gets re-arranged. But we did it anyway. Please let us <a target="_blank" rel="noopener noreferrer" href="https://docs.goauthentik.io/docs/">know what you think of the new Docs structure!</a>',
```

Comment:
Oh! I was thinking "But we did it anyway, and we think the new layout is a lot smarter and easier to navigate.  Let us know what you think."  But the banner itself is unremarkable, so approved.